### PR TITLE
Swatantra

### PR DESCRIPTION
**Pull request by Swatantramishra1**

### CODE REVIEW

File: String/longest_common_prefix/longest_common_prefixes_Swatantra.js [hunk 1]
```diff
@@ -1,72 +1,114 @@
...
-    let str2 = commonPrefix(arr, mid + 1, high);
+// if (ans.length) {
+//   console.log("Longest Common Prefix is = ", ans.join(""));
+// } else {
+//   console.log("There No Longest Common Prefix");
+// }
 
-    return commonPrefixUtil(str1, str2);
+const input = ["swatantra", "swapnil", "swastik"];
+
```
Comment (nidhihooda) [inline String/longest_common_prefix/longest_common_prefixes_Swatantra.js:75]: What is this method used for? Why is it required in the final push?

File: String/longest_common_prefix/longest_common_prefixes_Swatantra.js [hunk 2]
```diff
@@ -1,72 +1,114 @@
...
-  }
+//   for (let i = 0, j = 0; i <= l1 - 1 && j <= l2 - 1; i++, j++) {
+//     if (str1[i] != str2[j]) break;
+//     result.push(str1[i]);
+//   }
 
-  return result;
-}
+//   return result;
+// }
```
Comment (nidhihooda) [inline String/longest_common_prefix/longest_common_prefixes_Swatantra.js:52]: Why is this code in comments part of this pull request?

File: String/longest_common_prefix/longest_common_prefixes_Swatantra.js (hunk 1 above)
Comment (Swatantramishra1) [reply String/longest_common_prefix/longest_common_prefixes_Swatantra.js:75]: this is for just input

File: String/longest_common_prefix/longest_common_prefixes_Swatantra.js (hunk 2 above)
Comment (Swatantramishra1) [reply String/longest_common_prefix/longest_common_prefixes_Swatantra.js:52]: wrote two method for the same problem

**Pull request merged**